lib/pip.js

```diff
@@ -167,7 +167,7 @@ function dockerPathForWin(options, path) {
 function getSlimPackageCommands(options, targetRequirementsFolder) {
   const folderPath = dockerPathForWin(options, targetRequirementsFolder);
   const stripCmd = [
-    `&& find ${folderPath} -name "*.so" -exec strip {} ;`,
+    `&& find ${folderPath} -name "*.so" -exec strip {} \\;`,
```

Comment:
Here noticed a bug - at some point removed the `\\`, which broke the whole script... Now it works